code/modules/mob/dead/observer/observer.dm

```diff
@@ -599,15 +599,29 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 	set category = "Ghost"
 	set name = "Server Hop!"
 	set desc= "Jump to the other server"
+	if(notransform)
+		return
 	if (alert(src, "Jump to server running at [global.cross_address]?", "Server Hop", "Yes", "No") != "Yes")
 		return 0
 	if (client && global.cross_allowed)
 		src << "<span class='notice'>Sending you to [global.cross_address].</span>"
+		new /obj/screen/splash(client)
+		notransform = TRUE
+		sleep(29)	//let the animation play
+		notransform = FALSE
 		winset(src, null, "command=.options") //other wise the user never knows if byond is downloading resources
-		client << link(global.cross_address)
+		client << link(global.cross_address + "?server_hop=[key]")
```

Comment:
This doesn't work how you think it does.

the ? bit added at the end of the address is sent to client/New() as its arg and finally to client/Topic if client/New() doesn't capture it (ours does for reasons i can't remember)

It never makes it to world/Topic.